### ISSUE TITLE
Loading of cffi-grovel is broken

### ISSUE THREAD

**Issue by snmsts** [ultralisp/ultralisp#1]
```
$ ros --version
roswell 18.9.10.94(4f852d3)

$ ros dist list
 quicklisp      2018-08-31 http://beta.quicklisp.org/dist/quicklisp.txt

$ ros dist add http://dist.ultralisp.org/
Downloading http://dist.ultralisp.org/
##########################################################################
Installing dist "ultralisp" version "20180912-200916".
Downloading http://dist.ultralisp.org/ultralisp/20180912-200916/releases.txt
##########################################################################
Downloading http://dist.ultralisp.org/ultralisp/20180912-200916/systems.txt
##########################################################################

$ ros -e "(ql:quickload :cffi-grovel)"
To load "cffi-grovel":
  Load 5 ASDF systems:
    alexandria babel cffi-toolchain trivial-features uiop
  Install 1 Quicklisp release:
    fukamachi-woo

debugger invoked on a SB-PCL::NO-APPLICABLE-METHOD-ERROR in thread
#<THREAD "main thread" RUNNING {10005505B3}>:
  There is no applicable method for the generic function
    #<STANDARD-GENERIC-FUNCTION QL-DIST:INSTALLEDP (3)>
  when called with arguments
    (NIL).
See also:
  The ANSI Standard, Section 7.6.6

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY   ] Retry calling the generic function.
  1: [ABORT   ] Give up on "cffi-grovel"
  2: [CONTINUE] Ignore runtime option --eval "(ros:run '((:eval\"(ros:quicklisp)\")(:eval \"(ql:quickload :cffi-grovel)\")(:quit)))".
  3:            Skip rest of --eval and --load options.
  4:            Skip to toplevel READ/EVAL/PRINT loop.
  5: [EXIT    ] Exit SBCL (calling #'EXIT, killing the process).

((:METHOD NO-APPLICABLE-METHOD (T)) #<STANDARD-GENERIC-FUNCTION QL-DIST:INSTALLEDP (3)> NIL) [fast-method]
0] 5

$ ros dist preference ultralisp 0
ultralisp      0

$ ros -e "(ql:quickload :cffi-grovel)" -q
To load "cffi-grovel":
  Load 1 ASDF system:
    cffi-grovel
; Loading "cffi-grovel"
.

```

**Comment by snmsts** [comment]
I'm not sure what's actually happening here. @svetlyak40wt could you reproduce this?

**Comment by svetlyak40wt** [comment]
I'll try.

**Comment by snmsts** [comment]
I don't have experience... but to see the first line
```
# project system-file system-name [dependency1..dependencyN]
```

cffi-column should be taken as system-name...

**Comment by svetlyak40wt** [comment]
Aha! Good catch!

I'll investigate how to fix it.

**Comment by svetlyak40wt** [comment]
Interesting, even after I've fixed a bug ultralisp/quickdist@b7deefc within quickdist, it generates
``
fukamachi-woo woo cffi-grovel alexandria cffi cffi-toolchain
``

And what is more strange, it generates few lines for woo (and some other projects too):

```
fukamachi-woo woo cffi alexandria babel trivial-features uiop
fukamachi-woo woo cffi alexandria babel trivial-features uiop
fukamachi-woo woo cffi alexandria babel trivial-features uiop
fukamachi-woo woo cffi-grovel alexandria cffi cffi-toolchain
```
I need to go deeper!

**Comment by svetlyak40wt** [comment]
I fixed the issue in [20180921-054637](http://dist.ultralisp.org/ultralisp/20180921-054637/systems.txt) version of ultralisp distrbution which uses quickdist, with this fix: https://github.com/ultralisp/quickdist/commit/7802cde75f9f92edc96db7b271c2c3622a4b76d8
